assets/jupyterLab/install-fixed.j2.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# ------------------------------------------------------------ #
+#                                                              #
+#   JUPYTER LAB INSTALLATION SCRIPT FOR EXABYTE.IO PLATFORM.   #
+#                                                              #
+#    DO NOT MODIFY THIS FILE! ADJUST "configure.sh" SCRIPT     #
+#    IF YOU NEED TO CUSTOMIZE YOUR JUPYTER LAB ENVIRONMENT.    #
+#                                                              #
+# ------------------------------------------------------------ #
+
+export PYTHONDONTWRITEBYTECODE=1
+
+# Load the fixed python environment for jupyterlab
+# This is a consistent set of packages installable in Python 3, obtained by:
+# > pip3 install "jupyterlab==3.0.3" "exabyte-api-client>=2020.10.19" numpy>=1.17.3 \
+# >  pandas>=1.1.4 matplotlib seaborn ase pymatgen rdkit-pypi "matminer==0.7.0" scikit-learn xgboost
+
+scratchdir="/scratch/$USER/$PBS_JOBID"
+envdir="$scratchdir/.env"
+jupyterlabdir="/export/compute/software/jupyterlab-envs/py-3.8.6-jl-{{ application.version }}/"
```

Comment:
In the web-app we still allow users to select 0.33.12 as an application version. This is somewhat misleading as any new jupyter workflow that is created using the 0.33.12 version will in fact run on the 3.0.3 version due to the fact that the `configure.sh` contains a `requirements.txt` file that then overwrites the initial 0.33.12 install with a 3.0.3 install. 

I kept the templating here as we discussed that we would like the system to be extensible in the future but it does have an edge case with 0.33.12.

Solutions - create a symlink from py-3.8.6-jl-0.33.12 to py-3.8.6-jl-3.0.3 using salt, remove 0.33.12 from `web-app/src/exabyte/imports/init_applications/registry.js` as it has already been deprecated in terms of new jobs and it seems unlikely that any users currently rely on 0.33.12.

Reply:
should be `/export/compute/software/python-applications/jupyterlab/py-3.8.6/3.0.3/`